src/index.ts

```diff
@@ -488,20 +541,26 @@ export class Bigtable {
     const adminOptions = Object.assign(
       {},
       baseOptions,
+      getUniverseDomainOptions(options, options.BigtableTableAdminClient),
       {
         servicePath:
           customEndpointBaseUrl ||
-          getDomain('bigtableadmin', options.BigtableClient),
+          getDomain('bigtableadmin', options, options.BigtableTableAdminClient),
       },
       options,
     );
     const instanceAdminOptions = Object.assign(
       {},
       baseOptions,
+      getUniverseDomainOptions(options, options.BigtableInstanceAdminClient),
       {
         servicePath:
           customEndpointBaseUrl ||
-          getDomain('bigtableadmin', options.BigtableClient),
+          getDomain(
+            'bigtableadmin',
+            options,
+            options.BigtableInstanceAdminClient,
+          ),
```

Comment:
This now supports the use case where the universe domain is provided in the handwritten client options as well as maintaining support for when the universe domain is provided just for the gapic client.

Reply:
Note that this should be BigtableInstanceAdminClient as this is a setting for the instance admin client.